# large -----------------------------------------------
# origin: [35.069387336800794, 128.57896450335267, 49.0] #hop
# [35.069356068205529, 128.57891931499569, 49.0] #others
# origin: [35.0695517, 128.5788733, 49.0] #docking

# autonomous_goal: [35.0696123, 128.5788811, 49.0]

waypoints: [
            [35.069481592976622, 128.57889197025276, 49.0], #left bottom
            [35.069618508613225, 128.57887159855932, 49.0], #mid up
            [35.069464319011999, 128.57896821820279, 49.0], #right bottom 
            [35.069558500130611, 128.57893036813357, 49.0], #right up
            [35.069387336800794, 128.57896450335267, 49.0] # mid bottom
          ]


boundary1: [
  [35.069356068205529, 128.57891931499569, 49.0], # left bottom
  [35.069657964817701, 128.5787989131625, 49.0], # left top
  [35.06968001815909, 128.57889555834939, 49.0], # right top
  [35.069387283657569, 128.57901513340735, 49.0] # right bottom
]



# small -----------------------------------------------
origin: [35.0692817, 128.578876, 49.0] # small left bottom

autonomous_goal: [35.0695981, 128.5787174, 49.0]

boundary2: [
  [35.0692817, 128.578876, 49.0], # left bottom
  [35.0695906, 128.5786724, 49.0], # left top
  [35.0696118, 128.578718, 49.0], # right top
  [35.0693012, 128.5789177, 49.0] # right bottom
]


# 5ho -----------------------------------------------
# origin: [37.450633, 126.652904, 45.0]

# waypoints: [
#             [37.450708, 126.652794, 45.0],
#             [37.450693, 126.65276, 45.0],
#             [37.450712, 126.652725, 45.0],
#             [37.450633, 126.652904, 45.0]
#           ]

# autonomous_goal: [37.450693, 126.65276, 45.0]

# docking_enterence: [37.450633, 126.652904, 45.0]
# station1: [37.4504967, 126.6529305, 45.0]
# station2: [37.4504998, 126.6529019, 45.0]
# station3: [37.4511531, 126.6560461, 45.0] # temp


# fountain -----------------------------------------
# origin: [37.4481655,126.6535769, 45.0]

# waypoints: [
#             [37.4481063, 126.6535479, 45.0],
#             [37.4480537, 126.6534697, 45.0],
#             [37.4481307, 126.6535153, 45.0],
#             [37.4480379, 126.6534913, 45.0]
#           ]

# autonomous_goal: [37.4482068, 126.6535778, 45.0] #[37.4482053, 126.6535542, 45.0] # [37.4481655, 126.6535769, 45.0]

# docking_enterence: [37.4482068, 126.6535778, 45.0] #[37.4481778, 126.653581, 45.0]
# station1: [37.4480452, 126.6534751, 45.0]
# station2: [37.4480703, 126.6534976, 45.0]
# station3: [37.4480911, 126.6535127, 45.0]